indico/web/client/styles/modules/_category.scss

```
@@ -342,98 +419,243 @@ ul.category-resource-qtip {
+    .event-date {
+      flex: none;
     }
 
-    .event-icons {
-      padding: 0 15px;
-      font-size: 1.2em;
-      white-space: nowrap;
-      color: $dark-gray;
-      max-width: 56px;
-      min-width: 56px;
+    .event-labels {
+      @extend %flex-row;
+      --flex-wrap: nowrap;
 
-      a,
-      a:hover {
-        color: $dark-gray;
+      a {
+        color: var(--text-secondary-color);
       }
     }
 
+    .favorite-event,
+    .download-event {
+      &::before {
+        @extend %icon;
+      }
+    }
+
+    .favorite-event[aria-hidden] {
+      opacity: 0;
+    }
+
+    .favorite-event span {
+      @extend %visually-hidden;
+    }
+
+    .favorite-event::before {
+      @extend .icon-star;
+      color: var(--tag-generic-color);
+    }
+
+    .download-event {
+      color: var(--text-secondary-color);
+    }
+
+    .download-event::before {
+      @extend .icon-calendar;
+    }
+
     .list-name a {
       font-size: 17px;
     }
 
-    .today {
-      font-size: 13px;
-      color: #444;
-      background-color: #f6e8bd;
-      border-radius: 5px;
+    .date {
+      flex: none;
+      text-align: right;
+      border-color: transparent;
+      border-style: solid;
+      border-width: 0.2em 0.5em;
+      border-radius: 0.4em;
+    }
+
+    .ongoing {
+      background-color: var(--attention-color);
+      border-color: var(--attention-color);
     }
 
     .protected {
-      font-weight: normal;
-      color: $dark-gray;
+      span {
+        @extend %visually-hidden;
+      }
+
+      &::before {
+        @extend %icon;
+        @extend .icon-shield;
+        color: var(--protected-color);
+      }
+    }
+
+    .new-event-label,
+    .hidden-event-label,
+    .custom-event-label {
+      @extend %tag;
+    }
+
+    .new-event-label {
+      --tag-surface-color: var(--tag-generic-color);
+    }
+
+    .hidden-event-label {
+      --tag-surface-color: var(--tag-disabled-color);
+    }
+
+    .custom-event-label {
+      @extend %flex-row;
+      --flex-wrap: none;
+
+      &[data-color='red'] {
+        background: #e00000;
+      }
+
+      &[data-color='orange'] {
+        background: #fd8e26;
+        --tag-text-color: #333;
+      }
+
+      &[data-color='yellow'] {
+        background: var(--attention-color);
+        --tag-text-color: #333;
+      }
+
+      &[data-color='olive'] {
+        background: #76780d;
+        --tag-text-color: #333;
+      }
+
+      &[data-color='green'] {
+        background: #498500;
+      }
+
+      &[data-color='teal'] {
+        background: #00807d;
+      }
+
+      &[data-color='blue'] {
+        background: #0071c7;
+      }
+
+      &[data-color='violet'] {
+        background: #693b97;
+      }
+
+      &[data-color='purple'] {
+        background: #ac51b8;
+      }
+
+      &[data-color='pink'] {
+        background: #f99ac0;
+        --tag-text-color: #333;
+      }
+
+      &[data-color='brown'] {
+        background: #7e563a;
+      }
+
+      &[data-color='gray'] {
```

```suggestion
      &[data-color='grey'] {
```

This uses the SUI color names, so we're stuck with that spelling of "gray"... :/

---

Ah crap, I keep tripping on that. :D